### PR TITLE
feat(create-mud): change `anvil` to create a block every two seconds

### PR DESCRIPTION
**Pull request by qbzzt**
- Better simulate the production environment
- The indexer works better with this

### CODE REVIEW

File: templates/phaser/mprocs.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ procs:
     shell: pnpm mud dev-contracts --rpc http://127.0.0.1:8545
   anvil:
     cwd: packages/contracts
-    shell: anvil --base-fee 0
+    shell: anvil --base-fee 0 -b 2
```
Comment (holic) [inline templates/phaser/mprocs.yaml:10]: to make this more readable:
```suggestion
    shell: anvil --base-fee 0 --block-time 2
```

File: templates/react-ecs/mprocs.yaml [hunk 2]
```diff
@@ -7,4 +7,5 @@ procs:
     shell: pnpm mud dev-contracts --rpc http://127.0.0.1:8545
   anvil:
     cwd: packages/contracts
-    shell: anvil --base-fee 0
+    shell: anvil --base-fee 0 -b 2
+
```
Comment (holic) [inline templates/react-ecs/mprocs.yaml:11]: no need for the extra line at the end
```suggestion
```

**Comment by holic** [comment]
can we update all the mprocs files with the same? examples, etc.

also need a changeset for this since it impacts the create-mud package

**Comment by qbzzt** [comment]
How do I make a changeset?

**Comment by qbzzt** [comment]
> can we update all the mprocs files with the same? examples, etc.
> 
> also need a changeset for this since it impacts the create-mud package

Done and done

File: .changeset/polite-dots-tell.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"create-mud": patch
+---
+
+feat(create-mud): change `anvil` to create a block every two seconds
```
Comment (holic) [inline .changeset/polite-dots-tell.md:5]:
```suggestion
Updated `anvil` args with two second block time to better reflect L2s
```

**Review by holic — APPROVED** [review]

**Pull request merged**